app/pb_kits/playbook/data/menu.yml

```diff
@@ -74,3 +74,4 @@ kits:
     - title_detail
   - user
   - user_badge
+  - project
```

Comment:
Can you put this in alphabetical order + add it to the type patterns section?

Reply:
This is alphabetical now. I am not sure what is `type patterns` but I think this is what you mean. Let me know if this is the wrong place :)
Changes here 2a431c6